cirrus/server/Dockerfile

```
@@ -15,12 +13,9 @@ RUN apt-get update -qqy \
 
 WORKDIR /application-services
 
+# Checkout application services at a locked commit
 RUN git clone https://github.com/mozilla/application-services.git .
-RUN if [[ -z "${as_version}" ]] ; then \
-    echo "No application-services version supplied, using latest commit." ; \
-    else \
-    git fetch ; \
-    fi
+RUN git checkout da9301f829de67149d50dce5b709aed42f1bfe1b
```

It may need a fetch before the checkout?
```suggestion
RUN git fetch && git checkout da9301f829de67149d50dce5b709aed42f1bfe1b
```

---

No it doesn't, it includes the history as part of the clone 👍

---

Oh maybe you're right, why does that work locally and not in CI....

---

Yeah you're right @jeddai but that passed on my local but not in CI, they're on the same Docker version, even after a local prune.  I wonder why that would be.... 🤔